pkg/standalone/uninstall.go

```diff
@@ -10,6 +11,11 @@ import (
 
 // Uninstall deletes all installed containers
 func Uninstall(uninstallAll bool, dockerNetwork string) error {
+	dockerInstalled := utils.IsDockerInstalled()
+	if !dockerInstalled {
```

Comment:
I think this should be best-effort in that even if docker is not there it should still perform parts of uninstall that are related to that.  In this case part unrelated is deleting a rundatafile.  So if docker is not there is skips the docker container related items, tries the other ones, then at the end fails and reports docker is not installed etc.  What do you think?

Reply:
Yeah, it makes sense! I missed `rundata.DeleteRunDataFile()`. I think this change would be even easier if https://github.com/dapr/cli/pull/333 is merged!